backend/remote-state/oss/backend.go

```diff
@@ -196,10 +193,23 @@ func assumeRoleSchema() *schema.Schema {
 					Description: "The permissions applied when assuming a role. You cannot use this policy to grant permissions which exceed those of the role that is being assumed.",
 				},
 				"session_expiration": {
-					Type:         schema.TypeInt,
-					Optional:     true,
-					Description:  "The time after which the established session for assuming role expires.",
-					ValidateFunc: validation.IntBetween(900, 3600),
+					Type:        schema.TypeInt,
```

Comment:
Drawing attention here, as it's the only refactor (vs. straight deletion) in this PR - `validation.IntBetween` is used by a couple of provisioners and nothing else. This is a pretty low-value change, all told. 

When I started I thought I'd be able to remove both funcs in helper/validate, not just one, so I wouldn't be sad to revert it.

Reply:
I think this is OK, since we want to eventually remove the use of helper/schema anyway